element-templates/RESTConnectorTemplate.json

```diff
@@ -127,6 +127,16 @@
         "name": "httpOutParameter"
```

Comment:
Should this be renamed to `httpOutputParameter` or maybe `httpBodyParameter` now that we have `httpStatusCodeParameter`? Or if we want to differentiate from te request, `httpResponseBodyParameter` and rename `httpPayload` to `httpRequestBody`.

And should they have the suffix of `Parameter` or `Variable`? they are names of process variables so I think the later is more correct (which should make it easier for someone else to understand later)

I'll work on a PR to this branch for that. Feel free to share any initial reactions though.